datalad/interface/make_memory_engram.py

```diff
@@ -0,0 +1,74 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""For now just a wrapper for Dataset.remember_state()
+
+"""
+
+__docformat__ = 'restructuredtext'
+
+import logging
+
+from datalad.support.constraints import EnsureStr
+from datalad.support.constraints import EnsureNone
+from datalad.support.exceptions import InsufficientArgumentsError
+from datalad.support.gitrepo import GitRepo
+from datalad.support.param import Parameter
+from datalad.distribution.dataset import Dataset
+from datalad.distribution.dataset import EnsureDataset
+from datalad.distribution.dataset import datasetmethod
+from datalad.utils import getpwd
+
+from .base import Interface
+
+lgr = logging.getLogger('datalad.interface.commit')
+
+
+class MakeMemoryEngram(Interface):
+    """see Dataset.remember_state()."""
```

Comment:
I wonder if `@borrow_doc` could be used. Remember that this line is used for generating overall --help so it is important to have it 'proper'